_posts/2025-02-01-next-scala-lts.md

```diff
@@ -0,0 +1,82 @@
+---
+category: announcement
+permalink: /news/next-scala-lts.html
+title: "Next Scala 3 LTS version"
+by: Tomasz Godzik
+---
+
+**TLDR;**
+
+sun.misc.Unsafe, re-used in lazy val implementation, will be removed in a future
+JDK one of the Scala 3 Next minors in 2025 Q4 and the next LTS will drop JDK 8
+support
+
+## Intro
+
+For the compiler’s second LTS version, the Scala 3 compiler team will be
+dropping support for JDK 8 and is considering which later JDK to use.
+
+## Reasons for the change
```

Comment:
The difference between "Reasons for the change" and "Advantages" seems unclear to me.

I can see that you're trying to tell it as a story: first we hit this `Unsafe` issue, then we started thinking about it more broadly. But I don't think that story is conveyed clearly. "Reasons for the change" suggests that the section will contain the reasons. But it doesn't, many of the reasons are in the next section.

Maybe it's as simple as changing "Reasons for the change" to "Immediate motivation" and "Advantages" to "Longer-term motivation"? Something like that?

Reply:
Good idea, I did `long term motivation` instead though, what do you think?

Reply:
LGTM